ui/src/main/java/com/dat3m/ui/result/ReachabilityResult.java

```diff
@@ -113,9 +125,9 @@ private void run(){
 
     private void buildVerdict(Result result){
         StringBuilder sb = new StringBuilder();
-        sb.append("Condition ").append(program.getAss().toStringWithType()).append("\n");
-        sb.append(program.getFormat().equals(LITMUS) ? result == FAIL ? "Ok" : "No" : result).append("\n");
-        verdict = sb.toString();
+        sb.append("\n").append("Condition ").append(program.getAss().toStringWithType()).append("\n");
+        sb.append(program.getFormat().equals(LITMUS) ? safetyViolationFound ? "Ok" : "No" : result).append("\n");
+        verdict += sb.toString();
```

Comment:
This method is the only one relying on `safetyViolationFound`, so it should be a parameter rather than a member variable. Also, why don't we just pass the `prover` object (or just the `model`) and let this method compute the whole verdict, as its name suggests?
It could check the flagged axioms, the safety violation and even liveness from the model and produce whatever it needs to produce.